docs/pages/appbars/index.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+
+  <DocsPageTemplate>
+
+    <DocsPageSection title="Overview" anchor="#overview">
+      <p>
+        App bars are used to provide users a means of navigation through pages in our applications. They allow access to top-level pages, navigate back to a specific page, or dismiss edit forms
+      </p>
+
+      <p>
+        We have four types of app bars: <em>primary bars</em>, <em>back arrow bars</em>, <em>form bars</em>, and <em>bottom bars</em>
+      </p>
+
+      <h3>Primary bars</h3>
+      <p>Primary bars are placed on top level pages in Kolibri. They may house subtabs that lead to different sections of the application</p>
+      <DocsShow>
+        <img class="modal-img" src="./primary-1.png">
+      </DocsShow>
+      <DocsShow>
+        <img class="modal-img" src="./primary-2.png">
+      </DocsShow>
+
+      <h3>Back arrow bars</h3>
+      <p>Back arrow topbars allow users to navigate back to the previous page</p>
+      <DocsShow>
+        <img class="modal-img" src="./back-1.png">
+      </DocsShow>
+
+      <h3>Form bars</h3>
+      <p>Form bars allows users to dismiss a full-screen form and land on the previous page</p>
+      <DocsShow>
+        <img class="modal-img" src="./form-1.png">
```

Comment:
Should we have specifications on when to use black color vs purple? Or is this something we're looking to deprecate? It looks like there's a spec for the content renderer without its black app bar

Reply:
We decided during documentation phase that black is just for pages with editing functionality (single-task pages)